src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/ImportExport.cs

```diff
@@ -122,7 +122,7 @@ public static void UnusualExponentImportExport()
             AssertKeyEquals(ref unusualExponentParameters, ref exported);
         }
 
-        [Fact]
+        [ConditionalFact(nameof(EphemeralKeysAreExportable))]
```

Comment:
Technically this is a different problem, but the exportability might also come into play, so this is fine.